### PR TITLE
Feature/update

### PR DESCRIPTION
**Pull request by bhavyaidr**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,6 +61,12 @@
             <artifactId>jetty-servlets</artifactId>
             <version>9.4.18.v20190429</version>
         </dependency>
```
Comment (amitmbm) [inline pom.xml:63]: Why did you add this dependency? 

CC : - @bhavya121999

File: src/main/java/com/indore/ElasticsearchConfig.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import org.hibernate.validator.constraints.NotEmpty;
 
 /**
- * <File Description>.
```
Comment (amitmbm) [inline src/main/java/com/indore/ElasticsearchConfig.java:6]: This comment isn't appropriate, its a config for elastic search host and port not for application host and port.

File: src/main/java/com/indore/resources/UserResource.java [hunk 3]
```diff
@@ -52,6 +53,12 @@ public Response createUser(JsonNode user) {
         }
     }
 
+    /**
```
Comment (amitmbm) [inline src/main/java/com/indore/resources/UserResource.java:56]: No need to mention `GET` as its evident from the annotation, just provide the description of what this API does.
For Ex:- API for user search etc.

File: src/main/java/com/indore/resources/UserResource.java [hunk 4]
```diff
@@ -94,5 +115,57 @@ public Response deleteUser(@PathParam("userId") String userId) {
...
```
Comment (amitmbm) [inline src/main/java/com/indore/resources/UserResource.java:118]: It's unsafe to pass the password as a path param as it would be visible in the URL and best practice to pass the password as part of request body.

File: src/main/java/com/indore/resources/UserResource.java [hunk 5]
```diff
@@ -94,5 +115,57 @@ public Response deleteUser(@PathParam("userId") String userId) {
...
+     log.error("Error getting user doc and error is {}", e);
+     return Response.serverError().build();
+     }
+
+     }*/
+
+
 
 }
+
```
Comment (amitmbm) [inline src/main/java/com/indore/resources/UserResource.java:164]: Remove empty lines.

File: pom.xml (hunk 1 above)
Comment (bhavyaidr) [reply pom.xml:63]: To add the CrossOriginFilter class to the Dropwizard environment..we need to add a dependency on Jetty’s “servlets” module .

**Comment by amitmbm** [comment]
Changed userprofile POJO, hence not valid.
